test/python/test_operation.py

```diff
@@ -610,6 +610,17 @@ def test_Atanh_gpu(self):
         np.testing.assert_array_almost_equal(tensor.to_numpy(result), XT, decimal=5)
         self.check_shape(dx.shape(), (3, 2))
 
+    def test_SoftPlus(self):
+        X=np.array([1.0,2.0,3.0,4.0,5.0,6.0]).reshape(3,2).astype(np.float32)
+        XT=np.log(np.exp(X) + 1)
+        x=tensor.from_numpy(X)
+        x.to_device(gpu_dev)
+
+        result=autograd.softplus(x)
+        dx=result.creator.backward(x.data)
```

Comment:
pls also test the correctness of the gradients.

Reply:
I added gradient test